Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,20 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  // console.log(tasks, hourlyRate);
+  return tasks
+    .map(task => task.duration / 60)
+    .filter(taskHour => taskHour >= 2)
+    .reduce((total, perHour) => total + perHour * hourlyRate, 0);
```

Comment:
The `perHour` variable is actually still a duration in hours. So I would name it something like `durationInHours`.

Reply:
Thank you for the suggestion. I've renamed the variable accordingly.